contracts/L2/gateway/L2Migrator.sol

```diff
@@ -298,7 +301,7 @@ contract L2Migrator is ManagerProxyTarget, L2ArbitrumMessenger, IMigrator {
             payable(delegator).transfer(_fees);
         }
 
-        emit StakeClaimed(delegator, delegate, _stake, _fees);
+        emit StakeClaimed(delegator, _delegate, _stake, _fees);
```

Comment:
I think it makes more sense to use the delegate from the snapshot as opposed to allowing this value to potentially be the new delegate specified by the caller since all other params are from the snapshot. If a new delegate is specified by the caller that will still be logged in the Bond event emitted by the BondingManager.